core/src/test/resources/log4j.properties

```diff
@@ -14,5 +14,8 @@
 # limitations under the License.
 ##
 
-log4j.rootLogger=OFF
+log4j.rootLogger=INFO, stdout
```

Comment:
You don't want tests to be too noisy when running them in your IDE, for example. `OFF` seems a bit extreme, but the right level for a given project varies so I'll defer to @ewencp as he knows the project better.